docker/conda/environments/cuda11.8_dev.yml

```diff
@@ -15,11 +15,12 @@
 
 name: morpheus
 channels:
-    - rapidsai
-    - nvidia/label/cuda-11.8.0
-    - nvidia
-    - nvidia/label/dev # For pre-releases of MRC. Should still default to full releases if available
-    - conda-forge
+  - rapidsai
+  - nvidia/label/cuda-11.8.0
+  - nvidia
+  - nvidia/label/dev # For pre-releases of MRC. Should still default to full releases if available
+  - conda-forge
+  - numba
```

Comment:
When installing locally, I have no problems ommitting this channel and I see no packages from the `numba` channel installed when I do include it. Do you see something different? I would prefer not to include this new channel since they provide many conflicting packages since as pip, scipy, setuptools, sphinx and the cudatoolkit. Its just asking for issues down the line when we suddenly get a different package than expecting.

Reply:
Removed the numba channel